packages/buildcop/README.md

```diff
@@ -1,4 +1,56 @@
-Buildcop
+# Buildcop
+
+The Build Cop Bot manages issues for failing tests.
+
+* If a test fails, the bot will open an issue for it.
+* If a test passes, the bot will close the corresponding issue.
+* If the test fails _again_, the bot will reopen the issue, mark it as flaky, then
+  stop commenting and leave it up to a human to close.
+* If someone closes the issue and the test fails _again_, the bot will reopen the
+  issue and leave it up to a human to close again.
+
+Issues or feature requests? Please
+[file them on this repo](https://github.com/googleapis/repo-automation-bots/issues/new).
+
+## Usage
+
+### Installation
+
+1. Install the bot on your repo. See https://github.com/apps/build-cop-bot/.
+   Issues will not start being filed until you finish the rest of the steps.
+
+   **Note**: if your repo is in `googleapis`, the bot is already installed.
+1. Create `sponge_log.xml` xUnit XML files with your test results. There can be
+   more than one `sponge_log.xml` file as long as they are in different
+   directories. They must be named `sponge_log.xml`.
```

Comment:
Other tools only require the file to end in `sponge_log.xml`. Is it possible for that to continue?

Reply:
Yes. Done.